cmd/grr/workflow.go

```diff
@@ -45,7 +45,9 @@ func listCmd(registry grizzly.Registry) *cli.Command {
 	}
 	var opts Opts
 	var isRemote bool
+	var isVerbose bool
 	cmd.Flags().BoolVarP(&isRemote, "remote", "r", false, "list remote resources")
+	cmd.Flags().BoolVarP(&isVerbose, "verbose", "v", false, "show more information about resources")
```

Comment:
Having both a `log-level` and a `verbose` flag makes be :cry: and :confused:

Reply:
They are totally different though. One shows different logs, this tells Grizzly to report more back to the user about the resources. Do you have any suggestions for a different flag name?

Reply:
Ideally, I'd keep only `log-level` (even though it usually is named `verbosity`, with different levels) and use a `format` flag where needed.

Verbosity representing the level of debugging info being printed out (ie: what is the CLI doing), and format controlling the format of the information being printed.

A bit like what `kubectl` does

Reply:
In this PR, I would skip the verbose flag entirely and always print the source.

Cleaning up the way logs and actual info is handled in grizzly sounds a bit too big to be done here :)